### ISSUE TITLE
publishing GAN parameters

### ISSUE THREAD

**Issue by danielshaowers** [loic-lb/Unsupervised-Nuclei-Segmentation-using-Spatial-Organization-Priors#1]
Thanks for sharing your preprints + weights. Could you also include the "G_losses_GAN.pkl" file in your drive link?

**Comment by loic-lb** [comment]
Dear Danielshaowers,

The "G_losses_GAN.pkl" file was in fact reffering to "G_losses_system" created during training. It was a typo left in my code, I apologize for the confusion it may induce. I will check if I can find these files for the different experiments and upload them on the drive.

You can however use the test.py file without using these files as I included

**Comment by loic-lb** [comment]
Dear Danielshaowers,

The "G_losses_GAN.pkl" file was in fact reffering to "G_losses_system" created during training. It was a typo left in my code, I apologize for the confusion it may induce. I will check if I can find these files for the different experiments and upload them on the drive.

You can however use the test.py file without using these files as I included

**Comment by loic-lb** [comment]
Dear Danielshaowers,

The "G_losses_GAN.pkl" file was, in fact, referring to "G_losses_system.pkl" created during training. It was a typo in my code. I apologize for the confusion it may induce. I will check if I can find these files for the different experiments and upload them on the drive.

You can use the test.py file without actually using these files if you use the pre-trained models I put on the drive. Indeed, I included a particular condition for this case. It will just load the pre-trained model without looking at the loss. I will add this point to the Readme.

**Comment by danielshaowers** [comment]
works perfectly, thank you!
